api/server.go

```diff
@@ -38,6 +38,7 @@ func (ws *SecureWebServer) Start() error {
 
 	v1 := ws.echo.Group("/v1")
 	v1.POST("/container/scan", createScan)
+	v1.GET("/container/scan/:image", showScans)
```

Comment:
I'd drop the `/container` from the endpoints.

Reply:
Make sense, i'll drop them.